MoreCyclopsUpgrades/Managers/CyclopsHUDManager.cs

```diff
@@ -77,7 +78,12 @@ internal bool Initialize(CyclopsManager manager)
 
             this.Manager = manager;
 
-            powerIconTextVisibility = Player.main.currentSub == this.Cyclops;
+            float maxHealth = ModConfig.Settings.MaxCyclopsHealth();
+
+            Cyclops.live.data.maxHealth = maxHealth;
+            Cyclops.live.health = Mathf.Min(maxHealth, Cyclops.live.health);
```

Comment:
This should probably be moved to a patch method in `SubRoot.Awake`.  
Be sure that it doesn't trigger the "damaged" state when changing.